scripts/convertFigmaPalette.ts

```diff
@@ -0,0 +1,65 @@
+import fs from 'fs';
+import path from 'path';
+import prettier from 'prettier';
+import yargs from 'yargs';
+import { hideBin } from 'yargs/helpers';
+
+(async () => {
+    const { argv } = yargs(hideBin(process.argv)).boolean('test') as any;
+    const isTesting = argv.test || false;
+
+    const serializeConfig = async (config: any, stringifySpaces?: number) => {
+        try {
+            return await prettier.format(
+                `const palette = ${JSON.stringify(config, null, stringifySpaces)};\n\nexport default palette;`,
+                { parser: 'babel' },
+            );
+        } catch (error) {
+            console.error(error);
+            process.exit(1);
+        }
+    };
+
+    const normalizeName = (name: string) =>
+        name
+            .replace(/\//g, ' ')
+            .replace(/[^a-zA-Z0-9 ]/g, '')
+            .split(' ')
+            .map((word, index) =>
+                index === 0 ? word.toLowerCase() : word.charAt(0).toUpperCase() + word.slice(1),
```

Comment:
There are changes in casing like for example `coinsBTCBold` -> `coinsBtcBold`, but not sure if any of those changed keys is actually used in colors.